contracts/cyberpunk/src/set_version.rs

```diff
@@ -0,0 +1,15 @@
+#[macro_export]
+macro_rules! set_wasm_version {
+    ($s1: expr, $s2: expr) => {
+        /// Private module with no stability guarantees
+        mod __cw5 {
+            use super::*;
+
+            const FULL: &str = const_str::concat!("/", $s1, "/", $s2);
+
+            #[link_section = "cw5"]
+            #[allow(unused)]
+            static AS_BYTES: [u8; FULL.len()] = const_str::to_byte_array!(FULL);
```

Comment:
As simple as that. I like it.

Reply:
The `used` attribute might be useful as well, to avoid this to disappear after optimisation: https://doc.rust-lang.org/beta/reference/abi.html#the-used-attribute